File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.js
function solution(n) {
    var answer= 0;
    var i= String(n % 3)
    
    while (n >= 3){
       n = Math.floor(n / 3); 
        i += String(n % 3) 
    }

    for (let j= 0; j< i.length; j++){
        answer+= Number(i[j])*3**(i.length-1-j);
    }
    
    return answer
}

File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.py
def solution(n):
    temp = ''
    while n > 0:
        temp += str(n % 3)
        n = n // 3
    
    answer = int(temp,3)
    # for i, num in enumerate(temp):
    # answer += int(num) * 3 ** (len(temp)- 1 - i) 
    return answer